docs/contributing.md

```diff
@@ -0,0 +1,68 @@
+# Contributors guide
+
+Contributions are welcome, and they are greatly appreciated!
+Every little bit helps, and credit will always be given.
+
+## Code of conduct
+
+Please read and abide by the [Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all interactions with the community.
+
+## Reporting bugs
+
+If you find a bug, please open an issue at <https://github.com/ansible-community/awx-operator-helm/issues>.
+
+Please also include the following information in your issue:
+
+- Your operating system name and the output of `helm show chart awx-operator/awx-operator`
+- Any details about your local setup that might be helpful in troubleshooting.
+- Detailed steps to reproduce the bug.
+
+## Writing code
+
+Want to fix a bug or implement a feature?
+Great!
+
+Look through the [list of issues](https://github.com/ansible-community/awx-operator-helm/issues) for bugs and features.
+You can filter bugs with the `bug` label and features with the `enhancement` label.
+Any issue that has the `help wanted` label is available to work on.
```

Comment:
Same, this is one one sentence, maybe use bullet points?

Reply:
i like the idea to use bullet points here

Reply:
Hmm. This will render as a paragraph, which is what I intend.

I'm not sure bullet points really work here tbh. Not totally opposed. Just not entirely convinced.

Bullet points present related but independent pieces of text as a list that has no particular order. They are good for organizing key pieces of information in a visual way that makes the text easier to scan. This, in turn, aids readability and helps users absorb information more quickly.

You could argue for bullet points here but overall the surrounding text is quite light. I don't think it's dense enough that readers need the visual space that bullet points bring to the text. It's a paragraph with three sentences, which is kind of the sweet spot. I'd also say that the three sentences here are not entirely independent of each other.

Reply:
keeping as-is wfm